### DISCUSSION TITLE
Ionic build ios product archive

### DISCUSSION THREAD

**Discussion by BrandonAxellRuiz** [ionic-team/capacitor#6652] [Q&A]
when i run in xcode archive so i can publish to store it gives me the following error
![353845615_9684486611576394_6621525358239154837_n](https://github.com/ionic-team/capacitor/assets/30245046/3610523e-9aff-4b14-a61f-098a2090009b)

**Comment by jcesarmobile** [comment]
answered on ionic forum
https://forum.ionicframework.com/t/ionic-build-ios-product-archive/234127/2
